include/mata/nfa/nfa.hh

```diff
@@ -434,28 +439,28 @@ inline OnTheFlyAlphabet create_alphabet(const Nfas&... nfas) {
  * @param[in] nfas Vector of NFAs to create alphabet from.
  * @return Created alphabet.
  */
-OnTheFlyAlphabet create_alphabet(const ConstAutRefSequence& nfas);
+OnTheFlyAlphabet create_alphabet(const std::vector<std::reference_wrapper<const Nfa>>& nfas);
```

Comment:
It would be nice if we somehow did not have to define all these variants.

Reply:
I made a note and will have a look at this in the future.